assets/js/blocks/cart-checkout/cart/frontend.js

```diff
@@ -6,17 +6,27 @@ import { withRestApiHydration } from '@woocommerce/block-hocs';
 /**
  * Internal dependencies
  */
-import Block from './block.js';
+import FullCart from './full-cart';
 import renderFrontend from '../../../utils/render-frontend.js';
 
-const getProps = () => {
-	return {
-		attributes: {},
+const isCartEmpty = false; // @todo check if the cart has some products
```

Comment:
Good placeholder for now. We can iterate on how this will work in a future pull.